File: challenges/4-fizz-buzz/javascript/ellenmariadev/fizzbuzz.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question(`Digite um número: `, (valor) => {
  for (i = 1; i <= valor; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
      console.log("FizzBuzz");
    } else if (i % 3 === 0) {
      console.log("Fizz");
    } else if (i % 5 === 0) {
      console.log("Buzz");
    } else {
      console.log(i);
    }
  }
  readline.close();
});

File: challenges/4-fizz-buzz/python/trustindestiny/fizz-buzz.py
x = int(input('Digite um número: '))


def fizzbuzz():
    for n in range(1, x + 1):
        if (n % 3 == 0) and (n % 5 == 0):
            print('fizzbuzz')
            continue
        if (n % 3 == 0):
            print('fizz')
        elif (n % 5 == 0):
            print('buzz')
        else:
            print(n)


fizzbuzz()
